### ISSUE TITLE
Some buttons are not label

### ISSUE THREAD

**Issue by Oreonan** [roundcube/roundcubemail#7321]
Hi,
some buttons of RoundCube 1.4.3 are not label for screenreaders.
This bug is present on buttons of top of pages (contact, option, mail and others), screenreader (NVDA) just say "Button".
Please add label or aria-label on this buttons ?
I precise I am using elastic skin.
Thanks.

**Comment by alecpl** [comment]
Could you be more specific? I don't see any label issues in Firefox Accessibility tool.

**Comment by alecpl** [comment]
No feedback.

**Comment by Oreonan** [comment]
Sorry for delay
I will try to am more precise:
When I navigate with arrow keys on the head of RoundCube with elastic skin, NVDA Say (in french): "Bouton créer un nouveau courriel", and for the 4 nexts buttons, he just say: "bouton", so I think there buttons have no labels.
